### DISCUSSION TITLE
Giscus do not show when self-hosting

### DISCUSSION THREAD

**Discussion by shenweiyan** [giscus/giscus#705] [Q&A] [answered]
I follow the tutorial of https://github.com/giscus/giscus/blob/main/SELF-HOSTING.md for self-hosting. 

I set up giscus website with github pages, after the deployment is complete, I open the github pages url in the browser, and I can see the information of the giscus homepage readme, but the giscus is not displayed.

It seems I am missing something important, but I have no idear about it. So please give me some help. Thanks.

**Comment by shenweiyan** [comment]
After I set up giscus website on [Vercel](https://vercel.com/), it said : An error occurred: giscus is not installed on this repository

**Comment by laymonage** [comment, marked as answer]
GitHub Pages won't work because the site uses SSR.

As for the error, where are you seeing that? On the homepage? If so, you need to install your self-hosted instance on your repository and change these configuration values according to your repository:
https://github.com/giscus/giscus/blob/75f1a57dc7412fc8c40a827829c9cccb3012a143/pages/index.tsx#L144-L157

**Reply by shenweiyan** [reply]
Thank you very much. According to your prompts, Giscus finally shows.
